compile/WINDOWS_INSTRUCTIONS.md

```diff
@@ -0,0 +1,59 @@
+### Steps to compile IRBEM x64 on Windows
+1. Download and install MSYS2 https://www.msys2.org/
+2. Following the installation steps and update the packages:
+```
+pacman -Syu
+```
+After packages are updated, the terminal will be closed.
+3. Run `MSYS2 MSYS` from the start menu and update the rest of the packages
+```
+pacman -Su
+```
+4. Install base-devel package (https://packages.msys2.org/group/base-devel)
+```
+pacman -S base-devel
+```
+5. Install mingw-w64-x86_64-toolchain package (https://packages.msys2.org/group/mingw-w64-x86_64-toolchain)
```

Comment:
I'm curious why this is broken out into multiple stages here, where the other instruction set combines these two stages.

Reply:
I added the note to the file that the steps can be combined.